stdlib/public/Darwin/os/os.m

```diff
@@ -33,7 +33,7 @@
     const void     *olp_mh;
     const void     *olp_pc;
     const char     *olp_format;
-    uint8_t         olp_data[0];
+    uint8_t        *olp_data;
```

Comment:
This isn't the same; it's a trailing array member.

Reply:
ah, crud; right, the trailing array member is 0-sized, as opposed to the pointer which is pointer sized.